bitnami/janusgraph/Chart.yaml

```diff
@@ -5,19 +5,21 @@ annotations:
   category: Database
   licenses: Apache-2.0
   images: |
+    - name: cassandra
+      image: docker.io/bitnami/cassandra:4.1.6-debian-12-r6
     - name: janusgraph
-      image: docker.io/bitnami/janusgraph:1.0.0-debian-12-r11
+      image: docker.io/bitnami/janusgraph:1.0.0-debian-12-r15
     - name: jmx-exporter
       image: docker.io/bitnami/jmx-exporter:1.0.1-debian-12-r6
     - name: os-shell
-      image: docker.io/bitnami/os-shell:12-debian-12-r28
+      image: docker.io/bitnami/os-shell:12-debian-12-r30
 apiVersion: v2
 appVersion: 1.0.0
 dependencies:
-- condition: storageBackend.cassandra.enabled
+- condition: cassandra.enabled
```

Comment:
Revert this change, the parameter `storageBackend.cassandra.enabled` is still the one to use.

Reply:
Good catch